### PR TITLE
fix(multiple select input): do not display doubled dots

### LINKED ISSUE #250: Fix double ellipsis for selectize
**How it works**
If you select option with long name, first ellipsis adds, after selection second option - second ellipsis adds and ignores our right-padding for chevron:

CleanShot 22 11 2024 at 11-01

**How it should work**
Second ellipsis shouldn't appear and text shouldn't ignore select's right-padding.

### PR DESCRIPTION
**Pull request by wrmk**
Closes #250.

copied logic from [internal project](https://github.com/toughbyte/hub/pull/15709)

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

**Comment by wrmk** [comment]
https://github.com/user-attachments/assets/1cf03972-25af-42a5-b085-3cb4822e96b4

File: app/assets/javascript/controllers/multiple_select_component_controller.js [hunk 1]
```diff
@@ -85,4 +91,40 @@ export default class extends SelectComponentController {
...
+
+    if (inputWidth <= firstItemWidth) {
+      $selectizeInput.css("text-overflow", "");
+      otherItems.forEach((item) => item.style.visibility = "hidden");
+    } else if (selectize.items.length > 1) {
+      $selectizeInput.css("text-overflow", "ellipsis");
+      [firstItem, ...otherItems].forEach((item) => item.style.visibility = "");
+    } else {
+      $selectizeInput.css("text-overflow", "");
+      [firstItem, ...otherItems].forEach((item) => item.style.visibility = "");
```
Comment (wrmk) [inline app/assets/javascript/controllers/multiple_select_component_controller.js:127]: difference than in internal project, to fix mistake with invisible first item.

how to repeat in the internal project:
 - select long name
 - select short name that will be invisible
 - deselect long name
 - short name will be invisible

**Review by Lyams — APPROVED** [review]

File: app/assets/javascript/controllers/multiple_select_component_controller.js (hunk 1 above)
Comment (Lyams) [reply app/assets/javascript/controllers/multiple_select_component_controller.js:127]: Would we like to fix this in an internal project?

Comment (wrmk) [reply app/assets/javascript/controllers/multiple_select_component_controller.js:127]: will create an issue

Comment (wrmk) [reply app/assets/javascript/controllers/multiple_select_component_controller.js:127]: [created pr](https://github.com/toughbyte/hub/pull/15900)

**Pull request merged**